optimade/server/routers/utils.py

```diff
@@ -274,7 +274,31 @@ def get_entries(
         # Deduce the `next` link from the current request
```

Comment:
Is there a way to design this, so that we can overwrite the `next` link from a method in `EntriesCollection`. You added some implementation for paging mechanisms, but there is still no way to customise it. You have a handle (`collection`) on `EntriesCollection` here. Could we move this to a class method `EntriesCollection.create_next_link(response, request, params)`? This way `ElasticCollection` could specialise its behaviour.

Reply:
Sounds good to me, I can have a go at this tomorrow.